src/Symfony/Component/OpenApi/README.md

```diff
@@ -0,0 +1,302 @@
+# Symfony OpenAPI
+
+The OpenApi component provides a user-friendly, object-oriented API to build
+OpenAPI specifications using PHP.
+
+## Sponsor
+
+The Symfony OpenApi component is [backed][1] by [Selency][2].
+
+Selency is a team of 80 people, each more committed than the last to
+promoting second-hand goods and doing something positive for our planet.
+Open-Source is perfectly aligned with our values, leading us to contribute
+back to the Symfony ecosystem through a series of components.
+
+Help Symfony by [sponsoring][3] its development!
+
+## Usage
+
+Symfony OpenApi implements the OpenApi standard in its version 3.1. It allows to build
+documentations using PHP objects, giving more flexibility and reusability to OpenApi
+definitions. All features of OpenApi 3.1 are supported.
+
+### Writing documentation
```

Comment:
This should not be in the readme itself but in the symfony-docs repo (that's what the symfony-docs PR item is for in the PR template, while you linked to an issue).

If you want to make this doc available in the PR during initial review of the component to make it easier, I suggest you to move it to a separate file (that will be removed from the PR before merging it), using the RST format (so that it will be ready to be moved to the docs repo).